pom.xml

```diff
@@ -693,6 +670,145 @@
             </properties>
         </profile>
 
+
+        <profile>
+            <id>java9plus</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <!-- Don't compile module-info.java, see java 9 profile -->
```

Comment:
This is the Java 9 profile, right?

Edit: Oh, maybe the 9 execution? Is this comment needed?

Reply:
Removing comment. I think the configuration and naming is clear.